### ISSUE TITLE
Missing ENV['HOST']

### ISSUE THREAD

**Issue by jackietheripper** [standardnotes/listed#6]
Hi,

https://github.com/standardnotes/listed/blob/185121563b72492d0e876d65a9b419bebbe8b2ba/app/controllers/authors_controller.rb#L54

I think this line should be updated to:

`redirect_to "#{ENV['HOST']}/usage/?secret_url=#{@secret_url}"`

By adding this users can put the extension in a subdir of their webspace if they just change the HOST env-variable to the right value.

Regards,
Jelko

**Comment by mobitar** [comment]
Good point. Are you looking to host this on your own server?

**Comment by jackietheripper** [comment]
Yes, it works so far. I've just changed this line and added tha HOST variable to the .env file.
I was looking for a possibility to share the notes that are saved on my server to other people.
